src/main/java/core/basesyntax/service/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private FruitStorageDao storageDao;
```

Comment:
According to the encapsulation principle, the storageDao should be private to prevent direct access to it from outside of the class. Good job following this principle.